DisplayBundle/Resources/public/block/ContentList/css/style.css

```diff
@@ -1,16 +1,18 @@
-#main .content-list {
+.content-list {
     margin-top: 30px;
 }
-#main .content-list div {
+
+.content-list div {
```

Comment:
not quite optimal performance-wise, isn't it possible to add some direct descendant selector ">" or specified class ?